src/netif.c

```diff
@@ -2613,6 +2634,14 @@ static void netif_lcore_init(void)
                             LCORE_JOB_LOOP, kni_lcore_loop, 0);
     }
 
+#ifdef CONFIG_ICMP_FWD_CORE
+    netif_jobs[6].role = LCORE_ROLE_FWD_WORKER;
+    snprintf(netif_jobs[6].job.name, sizeof(netif_jobs[4].job.name) - 1, "%s", "icmp_forward_proc");
+    netif_jobs[6].job.func = icmp_forward_proc;
+    netif_jobs[6].job.data = NULL;
+    netif_jobs[6].job.type = LCORE_JOB_LOOP;
+#endif
+
```

Comment:
This feature is not part of `netif` module.  Please use `dpvs_lcore_job_register` to register the job within `icmp` module.

Reply:
Has been modified, use dpvs_lcore_job_register to register the job within icmp.